commands/env.go

```diff
@@ -46,12 +46,26 @@ func (o out) Println(a ...interface{}) {
 	_, _ = fmt.Fprintln(o, a...)
 }
 
+func getCWD(cmd *cobra.Command, args []string) (string, error) {
```

Comment:
This code allows an alternate CWD during testing.  Since the production code doesn't pass a context with a value, there's not much risk to the existing functionality.

Reply:
also, you don't need `args` in here I believe